packages/common/amplify_db_common/pubspec.yaml

```diff
@@ -11,7 +11,7 @@ environment:
 
 dependencies:
   amplify_db_common_dart: ^0.1.0
-  drift: ^2.2.0
+  drift: ^2.0.0
```

Comment:
why is this changed here/other places?

Reply:
The changes are from this commit https://github.com/aws-amplify/amplify-flutter/pull/2331/commits/be12ace6c3b71009b9df523c9727558313979b33